frontend/src/components/Modal/ResourceModal/index.jsx

```diff
@@ -3,41 +3,41 @@ import { useForm } from "react-hook-form";
 import { connect } from "react-redux";
 import { Button } from "reactstrap";
 import { openModal, closeModal } from "../../../redux/actions/modal";
-import { modalEnum, resourceEnum } from "../../../utils/enums";
+import { modalEnum } from "../../../utils/enums";
 import {
   editAndRefreshResource,
   addAndRefreshResource,
-  deleteAndRefreshResource
+  deleteAndRefreshResource,
 } from "../../../utils/api";
 import {
   currentResourceSelector,
-  isAddingResourceSelector
+  isAddingResourceSelector,
 } from "../../../redux/selectors/modal";
 import LAHModal from "../../Modal";
 import "../styles.scss";
 
-const ResourceModal = props => {
+const ResourceModal = (props) => {
   const { register, handleSubmit, errors } = useForm();
   const [deleteClicked, setDeleteClicked] = useState(false);
 
-  const onSubmit = data => {
+  const onSubmit = (data) => {
     props.isAddingResource ? handleAddResource(data) : handleEditResource(data);
   };
 
-  const handleEditResource = async data => {
+  const handleEditResource = async (data) => {
     data.tags = data.tags
       .split(",")
-      .filter(t => t)
-      .map(tag => tag.trim());
+      .filter((t) => t)
+      .map((tag) => tag.trim());
     await editAndRefreshResource(data, props.resource._id);
     props.closeModal();
   };
 
-  const handleAddResource = async data => {
+  const handleAddResource = async (data) => {
     data.tags = data.tags
       .split(",")
-      .filter(t => t)
-      .map(tag => tag.trim());
+      .filter((t) => t)
```

Comment:
Whenever we edit a file, the git hook changes everything in the file to the new Prettier 2.0 styling changes, so that's why there are all these random ones here and there.